include/boost/format/group.hpp

```diff
@@ -34,9 +34,15 @@ namespace detail {
 
 
 // empty group, but useful even though.
-struct group0 
+struct group0
 {
-    group0()      {}
+    BOOST_FORCEINLINE group0() {}
+#if !defined(BOOST_NO_CXX11_DELETED_FUNCTIONS)
```

Comment:
What compiler of the supported ones doesn't have deleted functions?

Reply:
This was prior to setting the minimum language level to C++11 due to dependencies doing the same.